scipy/linalg/_decomp_update.pyx.in

```diff
@@ -1718,15 +1718,15 @@ def qr_insert(Q, R, u, k, which='row', rcond=None, overwrite_qru=False, check_fi
     --------
     >>> from scipy import linalg
     >>> a = np.array([[  3.,  -2.,  -2.],
-                      [  6.,  -7.,   4.],
-                      [  7.,   8.,  -6.]])
+    ...               [  6.,  -7.,   4.],
+    ...               [  7.,   8.,  -6.]])
     >>> q, r = linalg.qr(a)
 
     Given this QR decomposition, update q and r when 2 rows are inserted.
                       
     >>> u = np.array([[  6.,  -9.,  -3.], 
-                      [ -3.,  10.,   1.]])
-    >>> q1, r1 = linalg.qr_insert(q, r, u, 2, 'row', False)
+    ...               [ -3.,  10.,   1.]])
+    >>> q1, r1 = linalg.qr_insert(q, r, u, 2, 'row')
```

Comment:
`rcond=False` does not pass through the validation, which only lets None through.